spec/classes/logstash_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'spec_helper'
```

Comment:
I can recommend these spec tests. In the puppet community (Vox Pupuli) we try to have extensive spec tests. and just a few acceptance tests (although we are always happy to get more tests).